docs/how-to/llm-fine-tuning-optimization/llm-inference-frameworks.rst

```diff
@@ -132,7 +123,7 @@ Installing vLLM
             ROCR_VISIBLE_DEVICES=0,1 python -m vllm.entrypoints.api_server --model /data/llama-2-7b-chat-hf --dtype float16 –tp 2 --port 8000 &
             ROCR_VISIBLE_DEVICES=2,3 python -m vllm.entrypoints.api_server --model /data/llama-2-7b-chat-hf --dtype float16 –tp 2--port 8001 &
 
-      6. To test, send it a curl request containing a prompt.
+      5. To test, send it a curl request containing a prompt.
 
          .. code-block:: shell
 
```

Comment:
below line 157: 164
1. To install the TGI Docker image, run the following commands.
   .. code-block:: shell
      # Install from Dockerfile
      git clone https://github.com/huggingface/text-generation-inference.git -b mi300-compat    
      cd text-generation-inference
      docker build . -f Dockerfile.rocm

->
1. To launch the TGI Docker instance, run the following commands.
   .. code-block:: shell
      # Launch the TGI Docker in the host machine
docker run --name tgi --rm -it --cap-add=SYS_PTRACE --security-opt seccomp=unconfined \
    --device=/dev/kfd --device=/dev/dri --group-add video --ipc=host --shm-size 256g \
    --net host -v $PWD:/data \
    --entrypoint "/bin/bash" \
    --env HUGGINGFACE_HUB_CACHE=/data \
    ghcr.io/huggingface/text-generation-inference:latest-rocm

Reply:
done